packages/@ember/-internals/glimmer/lib/component-managers/curly.ts

```diff
@@ -6,6 +6,7 @@ import {
 } from '@ember/-internals/owner';
 import { enumerableSymbol, guidFor } from '@ember/-internals/utils';
 import { addChildView, setElementView, setViewElement } from '@ember/-internals/views';
+import type { Option } from '@ember/-internals/utility-types';
```

Comment:
The type was just renamed `Nullable` upstream it seems? We should use that rather than duplicating it here

Reply:
Nullable is not exported.

This seems like a too-simple type to let another project keep maintenance of.

When we combine ember-source and glimmer-vm, we can unify these concepts in a `*/utility-types` package or something.

Reply:
It probably should be exported 🤷🏼‍♂️ it is simple, yes, but if it is (and probably it is) used in the signatures of public functions, etc, then it should be exported publicly too. But anyway, for now it is fine to have a copy here, but we should probably at least update to match its upstream name to avoid confusion.

Reply:
done!